### ISSUE TITLE
Support population "groups"

### ISSUE THREAD

**Issue by openjck** [mozilla/ensemble#23]
This is something that the Hardware Report currently supports for the *GPU Model* chart. There are three buttons: *INTEL*, *NVIDIA*, and *AMD*. Clicking one of them hides the population lines corresponding to that manufacturer.

![Screenshot of the GPU Model chart and population buttons on the Firefox Hardware Report](https://user-images.githubusercontent.com/933396/36236707-95a46842-11c5-11e8-805c-b3fecb39f6db.png)

This can be specified along with other metadata in an [ensemble-transposer](https://github.com/mozilla/ensemble-transposer) manifest. I'm not opposed to putting this metadata in the source dataset, but so far all metadata has gone into transposer manifests.

In other words, don't worry about how to format this. If you tell me what the groups are, I can set them up.
